AndroidManifest.xml

```diff
@@ -66,6 +66,18 @@
             </intent-filter>
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit